client/src/components/messages/SendMessage.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { ImAttachment } from "react-icons/im";
+import { MdSend } from "react-icons/md";
+
+const SendMessage = () => {
+  return (
+    <div className="flex items-center w-full absolute bottom-0 overflow-x-hidden bg-slate-50 p-2">
+      <ImAttachment />
```

Comment:
If we pass this the prop color="#265B8B" it'll take the primary color we've been using and make it more consistent with the overall color palette

Reply:
Maybe you can do this so I can checkout how it looks